crates/wit-parser/src/lib.rs

```diff
@@ -314,24 +346,27 @@ pub struct Interface {
     ///
     /// Export names are listed within the types themselves. Note that the
     /// export name here matches the name listed in the `TypeDef`.
+    #[serde(serialize_with = "serialize_id_map")]
     pub types: IndexMap<String, TypeId>,
 
     /// Exported functions from this interface.
     pub functions: IndexMap<String, Function>,
 
     /// The package that owns this interface.
+    #[serde(serialize_with = "serialize_optional_id")]
     pub package: Option<PackageId>,
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Serialize)]
 pub struct TypeDef {
     pub docs: Docs,
     pub kind: TypeDefKind,
     pub name: Option<String>,
     pub owner: TypeOwner,
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Serialize)]
+#[serde(rename_all = "lowercase")]
```

Comment:
Instead of `"lowercase"` could `"kebab-case"` be used for this and all other serialization keys?

Reply:
Does BA have a style guide or doc somewhere on how to represent names and identifiers? e.g. when to use kebab-case vs camel vs snake / screaming snake?

(given most target languages don’t support kebab-case for identifiers)

Reply:
Heh I don't think anything is anywhere near official to have these sort of guidelines at a BA level by any means. 

In any case though I figure kebab-case might be good to have since it matches the component model.

Reply:
Turns out all of the fields are single words, so kebab and lowercase are identical. Revisit if/when this changes?